packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Onboarding page
+</div>
```

Comment:
**nitpick:** Can you remove the files if the development is in other PR.

Reply:
Removed those files: https://github.com/ovh/manager/pull/8146/commits/99340aeadb8cc44b68c4acfef8174352be2029ea